lib/gsup/controller/ulr.py

```diff
@@ -92,7 +92,8 @@ async def __send_isd_request(self):
                            .with_msg_type(MsgType.INSERT_DATA_REQUEST)
                            .with_ie('imsi', self.__subscriber_info.imsi)
                            .with_msisdn_ie(self.__subscriber_info.msisdn)
-                           .with_ie('destination_name', '')
+                           .with_ie('cn_domain', 2)
```

Comment:
You're also removing destination name and adding cn_domain which seems unrelated to this commit.
Maybe move this modification into an own commit?